Code/Tools/AssetProcessor/assetprocessor_static_files.cmake

```diff
@@ -50,6 +50,7 @@ set(FILES
     native/FileProcessor/FileProcessor.h
     native/FileWatcher/FileWatcher.cpp
     native/FileWatcher/FileWatcher.h
+    native/FileWatcher/FileWatcherBase.h
```

Comment:
If you intend to merge this PR with a merge commit, and not a squash, then this commit isn't really correct. `FileWatcherBase.h` was added in 75aaa3c5fd78f5cc015a83e76cc7ff9c84015ded, but not added to the `assetprocessor_static_files.cmake` until this later commit.